bigquery/system-test/sync_query.test.js

```diff
@@ -0,0 +1,56 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var example = require('../sync_query');
+var querySpy = sinon.spy(example.bigquery, 'query');
```

Comment:
I don't think you need to do any spying in the system tests. Just inspect the response to verify that the API was hit and responded with expected data.